.github/workflows/build-and-test.yml

```diff
@@ -33,13 +33,16 @@ jobs:
     - name: Build the mod
       run: ./gradlew build
 
-    - name: Run server for 1 minute
+    - name: Set repository owner and name
       run: |
-        mkdir run
-        echo "eula=true" > run/eula.txt  
-        timeout 60 ./gradlew runServer || true
+        echo "REPOSITORY_OWNER=${GITHUB_REPOSITORY%/*}" >> $GITHUB_ENV
+        echo "REPOSITORY_NAME=${GITHUB_REPOSITORY#*/}" >> $GITHUB_ENV
+
+    - name: Publish package
+      run: ./gradlew publish -Pgroup=com.github."$REPOISITORY_OWNER" -Pversion=0.0.1-publish-test
```

Comment:
I would only publish a to any maven when a version tag is set. As a way of approving a commit for maven

Reply:
draft, will remove later